pylint/checkers/exceptions.py

```diff
@@ -333,19 +334,39 @@ def _check_raise_missing_from(self, node: nodes.Raise) -> None:
         # like `exc.with_traceback(whatever)`. We won't analyze these, we'll just assume
         # there's a violation on two simple cases: `raise SomeException(whatever)` and `raise
         # SomeException`.
+
         if containing_except_node.name is None:
             # The `except` doesn't have an `as exception:` part, meaning there's no way that
             # the `raise` is raising the same exception.
-            self.add_message("raise-missing-from", node=node)
-        elif isinstance(node.exc, nodes.Call) and isinstance(node.exc.func, nodes.Name):
-            # We have a `raise SomeException(whatever)`.
-            self.add_message("raise-missing-from", node=node)
+            class_of_old_error = "Exception"
```

Comment:
```suggestion
            class_of_old_error: str
```

If this crashes we have found a new pattern for which we should create an `elif`.

Reply:
It crashes for 
```python
try:
    int("9a")
except:
    raise ValueError('Invalid integer')
```
``Exception`` felt like a nice default value that handled broad-except and raise-missing-from with one message.

Reply:
Ah yeah. We could suggest `except ... as exc` as well but I guess `except Exception as exc` is fine.